.github/workflows/ci.yaml

```diff
@@ -126,12 +126,55 @@ jobs:
         if: matrix.os == 'macos-latest'
         run: export WASP_E2E_TESTS_SKIP_DOCKER=1
 
-      - name: Run tests
-        run: cabal test
+      # - name: Run tests
+      #   run: cabal test
 
-      - name: Ensure todoApp builds
-        if: matrix.os == 'ubuntu-20.04' || matrix.os == 'macos-latest'
-        run: ./tools/ensure_todoapp_builds.sh
+      # - name: Ensure todoApp builds
+      #   if: matrix.os == 'ubuntu-20.04' || matrix.os == 'macos-latest'
+      #   run: ./tools/ensure_todoapp_builds.sh
+
+      - name: Headless - Cache Node Modules
+        uses: actions/cache@v3
+        if: matrix.os == 'ubuntu-20.04'
+        with:
+          path: ~/.npm
+          key: node-modules-${{ hashFiles('./headless-test/package-lock.json') }}
```

Comment:
Should we preface all keys with `headless-` maybe? I imagine collisions with other steps are near impossible, but possibly just for identification's sake?

Reply:
Also, the logs say `Cache restored from key: node-modules-` but not the hash. Is that a possible bug?

Reply:
https://github.com/wasp-lang/wasp/actions/runs/4893035316/jobs/8735506230#step:14:16